helm/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 appVersion: "1.0.0"
-name: aws-efs-csi-driver
 description: A Helm chart for AWS EFS CSI Driver
-version: 0.3.0
+name: aws-efs-csi-driver
+version: 0.3.1
```

Comment:
Since values that used to work in the templates are no longer being used the version should go to 0.4.0 if the project isn't ready to move to 1.0.0

Reply:
I agree on changing to 0.4.0, it is a lot of changes, I don't think I can make the call to 1.0.0.